lua/ide/lib/async_client.lua

```diff
@@ -1,7 +1,7 @@
 local uv = vim.loop
 local logger = require('ide.logger.logger')
 
-Git = {}
+local Git = {}
```

Comment:
One touch up, this is another copy paste problem. Should be called just "Client"

Reply:
fixed